.github/workflows/push_pr.yml

```diff
@@ -18,6 +18,11 @@
 
 name: Build flink-connector-pulsar
 on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.head_ref || github.ref_name }}
```

Comment:
Probably a good idea to include `github.workflow` as well, if we ever add more workflows for PRs/commits.